sep490-infrastructure/keycloak-testcontainer-provider/build.sh

```
@@ -0,0 +1,10 @@
+podman manifest rm thongdh3401/keycloak:24.0.5
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)